Makefile

```diff
@@ -18,7 +18,12 @@ include cfg/Config.mk
 debug ?= none
 net ?= none
 merge_sections ?= yes
-bootloader ?= grub
+uefi ?= no
+ifeq ($(uefi), yes)
```

Comment:
i'm thinking it may be cleaner to define a `boot_spec` variable or something, with possible values of `uefi` or `bios`, that can be used in the rest of the makefile; that'd remove a lot of the many hard-to-read conditionals throughout.